WordPress/Sniffs/WP/DeprecatedParametersSniff.php

```diff
@@ -0,0 +1,317 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check for usage of deprecated parameters in WP functions and provide alternative based on the parameter passed.
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.11.0
+ */
+class WordPress_Sniffs_WP_DeprecatedParametersSniff extends WordPress_AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'wp_deprecated_parameters';
+
+	/**
+	 * Minimum WordPress version.
+	 *
+	 * This sniff will throw an error when usage of deprecated parameters is
+	 * detected if the parameter was deprecated before the minimum supported
+	 * WP version; a warning otherwise.
+	 * By default, it is set to presume that a project will support the current
+	 * WP version and up to three releases before.
+	 * This variable allows changing the minimum supported WP version used by
+	 * this sniff by setting a property in a custom phpcs.xml ruleset.
+	 *
+	 * Example usage:
+	 * <rule ref="WordPress.WP.DeprecatedParameters">
+	 *  <properties>
+	 *   <property name="minimum_supported_version" value="4.4"/>
+	 *  </properties>
+	 * </rule>
+	 *
+	 * @since 0.11.0
+	 *
+	 * @var string WordPress versions.
+	 */
+	public $minimum_supported_version = 4.4;
+
+	/**
+	 * Array of function, argument, and replacement function for deprecated argument.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @var array <string function name> => <array target parameter positions>
+	 */
+	protected $target_functions = array(
+
+		'get_last_updated' => array(
+			1 => array(
+				'value'   => '',
+				'version' => '0.0',
```

Comment:
Turns out `xfn_check()` was deprecated in version 2.5. The version in core was incorrect. https://core.trac.wordpress.org/ticket/39737